lib/main.dart

```diff
@@ -15,14 +21,24 @@ class MyApp extends StatelessWidget {
 
   @override
   Widget build(BuildContext context) {
-    final router = createAppRouter();
+    final authRepository = AuthRepository();
 
-    return MaterialApp.router(
-      locale: TranslationProvider.of(context).flutterLocale,
-      supportedLocales: AppLocaleUtils.supportedLocales,
-      localizationsDelegates: GlobalMaterialLocalizations.delegates,
-      routerConfig: router,
-      theme: appTheme,
+    return BlocProvider(
+      create: (context) => AuthBloc(authRepository)..add(CheckTokenRequested()),
```

Comment:
:upside_down_face: Is this needed?
```suggestion
      create: (context) => AuthBloc(authRepository).add(CheckTokenRequested()),
```

Reply:
Thanks, TIL :bulb: